README.md

```diff
@@ -1,318 +1,439 @@
 
-NSLog(@"%@",filteredCharacterArrayWithDictionaries);
+for (NSDictionary *character in liNames) {
+    NSLog(@"%@ spells his name with an 'li'.", character[@"name"]);
+}
 ```
+This will print, in some order:
 
+```
+Balin spells his name with an 'li'.
+Dwalin spells his name with an 'li'.
+```
+Conversely, if we instead replace the first `*` with `??`, the predicate will look for names with exactly two characters preceding the "li" and any substring following it:
 
-The output of this code will result in the following output in our debug console, as all of the below names contain the letter "a".
+```objc
+NSPredicate *liNamesPredicate = [NSPredicate predicateWithFormat:@"name like '??li*'"];
+NSArray *liNames = [middleEarthers filteredArrayUsingPredicate:liNamesPredicate];
 
+for (NSDictionary *character in liNames) {
+    NSLog(@"%@ spells his name with an 'li'.", character[@"name"]);
+}
+```
+This will print, in some order:
 
-######Output
 ```
-2014-10-20 16:49:29.111 filteringTests[36700:1987740] (
-        {
-        age = 39;
-        name = Mary;
-    },
-        {
-        age = 24;
-        name = Margaret;
-    },
-       {
-        age = 21;
-        name = Joshua;
-    }
-)
+Balin spells his name with an 'li'.
+Fíli spells his name with an 'li'.
+Kíli spells his name with an 'li'.
 ```
+Notice that Dwalin's name no longer appears since his name has three characters preceding the "li". Be careful when selecting which wildcard to use—you may not get the results you expect if you don't think through your wildcards.
 
-##### `LIKE`
+**Advanced:** *The* `LIKE` *comparator is a light form of checking against a regular expression, or "RegEx"; the* `MATCHES` *comparator is used for full RegEx functionality.*
 
-Ensures the left hand expression is "like" the right hand expression. 
+#### String Comparator Modifiers: `[c]`, `[d]`, and `[cd]`
 
-Okay, so that is not particularly self-explanatory is it? The `LIKE` keyword allows us to take advantage of regular expression matchers `*` and `?`. Here is an example, but if you don't understand this just yet, feel free to come back to it after you read up on regular expressions.
+*Namárië! Nai hiruvalyë Valimar.  
+Nai elyë hiruva. Namárië!*  
+—from *The Lord of the Rings* by J.R.R. Tolkien
 
+These string comparators can also be modified to be insensitive to case (`[c]`), insensitive to diacritical characters (`[d]`), or insensitive to both (`[cd]`). These options, written inside square brackets `[``]`, can be appended to the selected string comparator within the predicate format string in the manner of `CONTAINS[cd]`. 
+
+Case insensitivity treats uppercase letters equally to lowercase letters. Diacritical insensitivity treats letters with a diacritical mark as their base letter; [diacritical marks][diacritical_marks] are letters with special accents, such as the ones in the above excerpt from *Galadriel's Lament* in *The Lord of the Rings* which is in the fictional elven language *Quenya*.
```

Comment:
I point out which characters you mean. There is so much stuff going on in that quote that I'm not sure what you mean. Simplify. "handles accented letters the same as unaccented letters. So "e" and ´é" are treated the same.